resources/public/api/conf/2.0/scenarios/list.md

```diff
@@ -31,6 +31,18 @@
             <td><p>UK_PROPERTY_MULTIPLE</p></td>
             <td><p>Simulates a successful response containing a mixture of uk-property-fhl and uk-property-non-fhl.</p></td>
         </tr>
+        <tr>
+            <td><p>FOREIGN_PROPERTY</p></td>
+            <td><p>Simulates a successful response containing a single foreign-property.</p></td>
+        </tr>
+        <tr>
+            <td><p>FOREIGN_PROPERTY_FHL_EEA</p></td>
+            <td><p>Simulates a successful response containing a single foreign-property-fhl-eea.</p></td>
+        </tr>
+        <tr>
+            <td><p>FOREIGN_PROPERTY_MULTIPLE</p></td>
+            <td><p>Simulates a successful response containing multiple foreign-properties.</p></td>
```

Comment:
Could this be changed to like the uk-property wording? Something like "containing a mixture of foreign-property and foreign-property-fhl-eea."